src/column.rs

```diff
@@ -409,10 +409,10 @@ impl HashColumn {
 		self.write_reindex_plan_locked(tables, reindex, key, address, log)
 	}
 
-	fn write_reindex_plan_locked<'a, 'b>(
+	fn write_reindex_plan_locked(
 		&self,
-		mut tables: RwLockUpgradableReadGuard<'a, Tables>,
-		mut reindex: RwLockUpgradableReadGuard<'b, Reindex>,
+		mut tables: RwLockUpgradableReadGuard<'_, Tables>,
+		mut reindex: RwLockUpgradableReadGuard<'_, Reindex>,
```

Comment:
Can they be just ignored? (or somehow having two _ indicate they are different)

Reply:
Yes, they can. I have pushed an extra commit in this PR removing them (and some others)

Reply:
Thanks :)